tests_metricflow/snapshots/test_query_output.py/str/DuckDB/test_scd_with_coarser_grain__query_output.txt

```diff
@@ -0,0 +1,8 @@
+test_name: test_scd_with_coarser_grain
+test_filename: test_query_output.py
+---
+metric_time__month      listing__capacity    family_bookings
+--------------------  -------------------  -----------------
+2020-01-01T00:00:00                     3                  2
+2020-01-01T00:00:00                     4                  5
+2020-01-01T00:00:00                     5                  2
```

Comment:
Ok. This one is also interesting because the input data is actually truncated by the filter. This is the behavior we're implementing here but I suspect it's worth documenting these cases in some detail when we update the docs page.